### ISSUE TITLE
can stop seeding datasets without deleting data

### ISSUE THREAD

**Issue by okdistribute** [datproject/dat.land-roadmap#22]

**Comment by okdistribute** [comment]
would this one be beta instead? Unless I have misinterpreted 'publish', wouldn't this require having a list of user accounts and associated published dats ?

**Comment by laurengarcia** [comment]
depends on how you look at it? kind of a product-y decision. imo i would like the option to unpublish dats whether i have an account or not. its really up to you guys tho. personally i could be persuaded either way.

**Comment by laurengarcia** [comment]
(am interpreting `publish` very loosely -- think drag and drop files as an anonymous (alpha) user)

**Comment by okdistribute** [comment]
Perhaps we need to define the difference between 'publish' and 'broadcast' -- are we talking about leaving the swarm? If so, makes sense to have that in there. 

I was thinking 'publish' was sort of like 'npm publish,' which would mean creating a user account/name, maybe some metadata, and pushing it to a mirror

**Comment by okdistribute** [comment]
Right, OK so you mean like, leaving the swarm! That makes sense for it to be in alpha then.

**Comment by laurengarcia** [comment]
yeah, was thinking of this as "unbroadcast" in your terminology. maybe i should change the wording in the title from "unpublish" to "remove". was thinking of this task in terms of "how does dat metadata get removed".

**Comment by laurengarcia** [comment]
another aspect of this task is: only the archive owner should be able to remove the dataset. but we should also probably have an admin tool in dat.land as well for removing inappropriate/spam dats.

**Comment by okdistribute** [comment]
Yeah, I agree. I'd say let's save the admin tool for until we have user accounts.  

How about if we make this issue for the smallest thing, which is that a user can remove a dat from the swarm. Similar to how in BitTorrent you can stop seeding.

**Comment by okdistribute** [comment]
@juliangruber is there a way to do this on the desktop app? (stop seeding)

**Comment by juliangruber** [comment]
@karissa you can remove a dat, but currently that only `rm -Rf`s its download path and makes sure when you launch the app for the next time it won't replicate it any more. i haven't added logic for closing the swarm yet

**Comment by okdistribute** [comment]
ok cool makes sense. we might want some sort of a 'pause' button on it then. maybe we could look at how bittorrent sync or utorrent does this and copy the ui
